.github/workflows/spellchecker.yml

```diff
@@ -0,0 +1,24 @@
+name: spellchecker
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  misspell:
+    name: runner / misspell
```

Comment:
extra space?

Reply:
Hmmm. I reference the PR https://github.com/apache/apisix/pull/3011.
It's harmless, right?

Reply:
ya, it's harmless, but what’s the reason to have spaces and slash for a name?

Reply:
OK, I will fix it.